Week-2/Session6-AdvCSS/answerAnimation.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Animating the answer</title>
+    <style>
```

Comment:
looks good, but try to keep css separate in another file and import that here to have separation of concern while working with html and css.